modules/tfm/tfm/boards/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif()
 
 if (${TFM_PARTITION_CRYPTO})
   target_link_libraries(platform_s PRIVATE platform_cc3xx)
-  target_link_libraries(platform_s PRIVATE crypto_service_mbedtls)
+  target_link_libraries(platform_s PRIVATE crypto_service_mbedcrypto)
```

Comment:
This was a workaround. You can remove the line.

Reply:
Removed the line. But I'm awaiting applying the patch so we don't cancel the CI run.